### DISCUSSION TITLE
"Our scrummy Project Managing" Blog post

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#26] [General]
Hey,
comment your feedback about our "Our scrummy Project Managing" blog post from your peer-review here:

**Comment by marina99999** [comment]
Dear Team FileFighter,

I like it that you explain all your steps really comprehensible.
To write a blogging tip of the day is a good idea and maybe my team will use the same tool in the next time too.
Thank you for the advice!

We look forward to hearing from you.

Your Team SweatForSuccess

**Comment by nyando** [comment]
Hey FileFighter Team,

looks like your setup is very similar to ours.
The YouTrack CLI is pretty neat, as a fellow command line enthusiast I appreciate the input.
Same goes for the link checker :)
I also enjoyed the commit names in your git history.

Keep up the good work!

The Socialyze Team
